ThreadJob/ThreadJob/PSThreadJob.cs

```diff
@@ -797,15 +806,6 @@ private void HookupHostDataDelegates(ThreadJobHost host)
             ThreadJobHostUI hostUI = host.UI as ThreadJobHostUI;
             System.Diagnostics.Debug.Assert(hostUI != null, "Host UI cannot be null.");
 
-            hostUI.Output.DataAdded += (sender, dataAddedEventArgs) =>

```

Comment:
This would be a breaking change for anyone depending on Write-Host in the Output stream.  So I think we should make this an option with a new cmdlet switch.  Another thing we could do is (optionally) write to the current PSHost, but that would be weird with multiple host output spew appearing while each job is run simultaneously, and is why I decided to add it to the Output stream.